README.md

```diff
@@ -1,32 +1,60 @@
 # Setup
 
-install docker. This is the hard part.
-
-## prerequisites
-
-- Virtualbox - this could probably change
-- helpful script (drop in your bashrc). sets env and echos the ip of the new container
+## Prerequisites
+
+zshell is recommended. The fish shell has been problematic.
+
+1. `cd app; npm install`
+1. Install Virtualbox (tested on 5.0.6)
+1. Install Docker (this is the hard part)
+    1. `brew install docker` OR download standalone installer
```

Comment:
should just be:

1. Follow install directions on [docker site](https://docs.docker.com/installation/mac/).